packages/beacon-node/src/execution/engine/interface.ts

```diff
@@ -38,6 +38,22 @@ export enum ExecutionEngineState {
   AUTH_FAILED = "AUTH_FAILED",
 }
 
+export enum ClientCode {
```

Comment:
where are those defined? we might wanna add a reference here. Do we need to define them at all in the code?

Reply:
> where are those defined? we might wanna add a reference here. Do we need to define them at all in the code?

It's defined here https://github.com/ethereum/execution-apis/blob/main/src/engine/identification.md . So there is a specific set of client code we need to read.

Will add a comment in the code for reference.

Reply:
yes pls jsdoc comment with link